src/unity/Assets/Crest-Examples/BoatDev/Scripts/BoatAlignNormal.cs

```diff
@@ -59,6 +59,15 @@ void FixedUpdate()
 
         // estimate water velocity
         Vector3 velWater = (_displacementToBoat - _displacementToBoatLastFrame) / Time.deltaTime;
+        if(OceanRenderer.Instance._createFlowSim) {
+            Vector2 surfaceFlow;
+            int lod  = LodDataFlow.SuggestDataLOD(new Rect(position.x, position.z, 0f, 0f), _boatWidth);
```

Comment:
we could pull out the lod up above and use it for sampling the other data, but its not required for this PR